control-plane/stor-port/src/transport_api/mod.rs

```diff
@@ -184,6 +184,12 @@ impl From<HostNqnParseError> for ReplyError {
     }
 }
 
+impl From<uuid::Error> for ReplyError {
+    fn from(error: uuid::Error) -> Self {
+        Self::invalid_argument(ResourceKind::Unknown, "uuid", format!("{error:?}"))
+    }
+}
```

Comment:
ResourceKind seems wrong, not sure you can do this as such

Reply:
Ill match result and return error. No need of this conversion.